src/pybind/mgr/cephadm/serve.py

```diff
@@ -725,6 +725,11 @@ def _check_daemons(self) -> None:
                     dd.daemon_type in CEPH_TYPES:
                 self.log.info('Reconfiguring %s (extra config changed)...' % dd.name())
                 action = 'reconfig'
+            if not action and dd.daemon_type in MONITORING_STACK_TYPES and not self.mgr.spec_store[dd.service_name()].deleted:
+                if dd.container_image_name != self.mgr._get_container_image(dd.name()):
+                    self.log.info('Redeploying %s. Daemon\'s current image does not match mgr/cephadm/container_image_%s config setting.'
+                                  % (dd.name(), dd.daemon_type))
```

Comment:
Maybe ``Redeploying %s (container image changed)...' % dd.name()`` so this is consistent with the other messages.

Reply:
nit, no need to use `%` for formatting the logging message, just
```py
self.log.info('Redeploying %s. Daemon\'s current image does not match mgr/cephadm/container_image_%s config setting.',
              dd.name(), dd.daemon_type)
```